Organization settings button does not work

#### Problem description

Organization settings button currently navigates to organization email settings page, which does not exist (since email settings exist only on platform level), and which should instead, by design, navigate to organization privacy settings page.

#### Replication Steps

- Go to `admin/organization/<organization ID>/projects` and click on setting button on top right
- It does not take you anywhere but gives the following error
```
Error: Could not resolve 'admin.organizations.settings.email' from state 'admin.organization.users'
    at Object.transitionTo (webpack:///./~/angular-ui-router/release/angular-ui-router.js?:3193:17)
    at Object.go (webpack:///./~/angular-ui-router/release/angular-ui-router.js?:3121:21)
    at setupState (webpack:///./src/app/index.run.js?:47:28)
    at eval (webpack:///./src/app/index.run.js?:58:13)
    at Scope.$broadcast (webpack:///./~/angular/angular.js?:18323:28)
    at Object.transitionTo (webpack:///./~/angular-ui-router/release/angular-ui-router.js?:3286:24)
    at Object.go (webpack:///./~/angular-ui-router/release/angular-ui-router.js?:3121:21)
    at eval (webpack:///./~/angular-ui-router/release/angular-ui-router.js?:4195:16)
    at eval (webpack:///./~/angular/angular.js?:19945:31)
    at completeOutstandingRequest (webpack:///./~/angular/angular.js?:6048:10) undefined
(anonymous) @ VM44 angular.js:14202
2VM44 angular.js:14202 Error: Could not resolve 'admin.organizations.settings.email' from state 'admin.organization.teams'
    at Object.transitionTo (webpack:///./~/angular-ui-router/release/angular-ui-router.js?:3193:17)
    at Object.go (webpack:///./~/angular-ui-router/release/angular-ui-router.js?:3121:21)
    at setupState (webpack:///./src/app/index.run.js?:47:28)
    at eval (webpack:///./src/app/index.run.js?:58:13)
    at Scope.$broadcast (webpack:///./~/angular/angular.js?:18323:28)
    at Object.transitionTo (webpack:///./~/angular-ui-router/release/angular-ui-router.js?:3286:24)
    at Object.go (webpack:///./~/angular-ui-router/release/angular-ui-router.js?:3121:21)
    at eval (webpack:///./~/angular-ui-router/release/angular-ui-router.js?:4195:16)
    at eval (webpack:///./~/angular/angular.js?:19945:31)
    at completeOutstandingRequest (webpack:///./~/angular/angular.js?:6048:10) undefined
```

#### Expected Behavior or Output

Organization settings button should lead you to organization privacy settings page.

--- a/app-frontend/src/app/index.routes.js
+++ b/app-frontend/src/app/index.routes.js
@@ -628,8 +628,7 @@ function adminStates($stateProvider) {
             },
             templateUrl: organizationSettingsTpl,
             controller: 'OrganizationSettingsController',
-            controllerAs: '$ctrl',
-            redirectTo: 'admin.organizations.settings.email'
+            controllerAs: '$ctrl'
         })
         .state('admin.platform', {
             title: 'Platform',

--- a/app-frontend/src/app/pages/home/home.html
+++ b/app-frontend/src/app/pages/home/home.html
@@ -55,7 +55,7 @@
             <p>
               Hello, <strong>{{$ctrl.authService.getProfile().nickname}}</strong>
             </p>
-            <a class="btn btn-light" ui-sref="settings.profile">Manage account</a>
+            <a class="btn btn-light" ui-sref="user.settings.profile">Manage account</a>
           </section>
           <section ng-if="$ctrl.blogPosts && $ctrl.blogPosts.length">
             <h5>From our blog</h5>

